apps/meteor/tests/data/livechat/department.ts

```diff
@@ -57,11 +57,23 @@ new Promise((resolve, reject) => {
 		});
 });
 
-export const createDepartmentWithAnOnlineAgent = async (): Promise<{department: ILivechatDepartment, agent: {
-	credentials: IUserCredentialsHeader;
+type OnlineAgent = {
 	user: WithRequiredProperty<IUser, 'username'>;
-}}> => {
-    const { user, credentials } = await createAnOnlineAgent();
+	credentials: IUserCredentialsHeader;
+};
+
+export const createDepartmentWithAnOnlineAgent = async (overrideUser?: OnlineAgent): Promise<{department: ILivechatDepartment, agent: OnlineAgent }> => {
```

Comment:
I think it would be better to create another function for this, something like `createDepartmentUsingExistingOnlineAgent`. WDYT?